eras/conway/impl/testlib/Test/Cardano/Ledger/Conway/Imp.hs

```diff
@@ -97,28 +97,20 @@ spec ::
   Spec
 spec = do
   BabbageImp.spec @era
-  describe "ConwayImpSpec - post bootstrap (protocol version 10)" $
-    withImpStateWithProtVer @era (natVersion @10) $ do
-      describe "BBODY" $ Bbody.spec @era
-      describe "DELEG" $ Deleg.spec @era
-      describe "ENACT" $ Enact.spec @era
-      describe "EPOCH" $ Epoch.spec @era
-      describe "GOV" $ Gov.spec @era
-      describe "GOVCERT" $ GovCert.spec @era
-      describe "UTXO" $ Utxo.spec @era
-      describe "UTXOS" $ Utxos.spec @era
-      describe "RATIFY" $ Ratify.spec @era
-      describe "LEDGER" $ Ledger.spec @era
-  describe "ConwayImpSpec - bootstrap phase (protocol version 9)" $
-    withImpState @era $ do
-      describe "BBODY" $ Bbody.spec @era
-      describe "CERTS" $ Certs.spec @era
-      describe "DELEG" $ Deleg.spec @era
-      describe "ENACT" $ Enact.relevantDuringBootstrapSpec @era
-      describe "EPOCH" $ Epoch.relevantDuringBootstrapSpec @era
-      describe "GOV" $ Gov.relevantDuringBootstrapSpec @era
-      describe "GOVCERT" $ GovCert.relevantDuringBootstrapSpec @era
-      describe "UTXO" $ Utxo.spec @era
-      describe "UTXOS" $ Utxos.relevantDuringBootstrapSpec @era
-      describe "RATIFY" $ Ratify.relevantDuringBootstrapSpec @era
-      describe "LEDGER" $ Ledger.spec @era
+  let
+    conwayImpSpec protVer =
+      describe ("ConwayImpSpec - " <> show protVer) $
+        withImpStateWithProtVer @era protVer $ do
+          describe "BBODY" $ Bbody.spec @era
+          describe "CERTS" $ Certs.spec @era
+          describe "DELEG" $ Deleg.spec @era
+          describe "ENACT" $ Enact.spec @era
+          describe "EPOCH" $ Epoch.spec @era
+          describe "GOV" $ Gov.spec @era
+          describe "GOVCERT" $ GovCert.spec @era
+          describe "LEDGER" $ Ledger.spec @era
+          describe "RATIFY" $ Ratify.spec @era
+          describe "UTXO" $ Utxo.spec @era
+          describe "UTXOS" $ Utxos.spec @era
+   in
+    forM_ [natVersion @9, natVersion @10] conwayImpSpec
```

Comment:
This will ensure that we are running for all protocol versions, just in case we have another intra-era hard fork
```suggestion
    forM_ [eraProtVerLow @Conway .. eraProtVerHigh @Conway] conwayImpSpec
```